fontc/src/workload.rs

```diff
@@ -385,6 +385,14 @@ impl Workload {
                 .get()
                 .difference(&preliminary_glyph_order)
             {
+                let id = AnyWorkId::Be(BeWorkIdentifier::GlyfFragment(glyph_name.clone()));
+                // bracket glyphs weren't in the perlim order, but because
+                // they exist in IR they get added along with the main BE glyph
+                // work.
+                if self.jobs_pending.contains_key(&id) || self.success.contains(&id) {
+                    log::debug!("job exists for {glyph_name}, skipping");
+                    continue;
+                }
```

Comment:
This bit surprises me, I'd have expected the existing logic to suffice.

Reply:
#1447 proposes doing this w/o modifying the core processing, keeping most of the bracket changes in glyphs2fontir

Reply:
Resolved by merging ^